<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartRemoteDialog</class>
 <widget class="QDialog" name="StartRemoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Debugger</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="debuggerLabel">
       <property name="text">
        <string>Debugger:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Utils::PathChooser" name="debuggerPathChooser" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="executableLabel">
       <property name="text">
        <string>Local executable:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="executablePathChooser" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="channelLabel">
       <property name="text">
        <string>Host and port:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="channelLineEdit">
       <property name="text">
        <string notr="true">localhost:5115</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="architectureLabel">
       <property name="text">
        <string>Architecture:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="architectureComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sysrootLabel">
       <property name="text">
        <string>Sysroot:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Utils::PathChooser" name="sysrootPathChooser" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="useServerStartScriptLabel">
       <property name="text">
        <string>Use server start script:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="useServerStartScriptCheckBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="serverStartScriptLabel">
       <property name="text">
        <string>Server start script:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Utils::PathChooser" name="serverStartScript" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
